trip_app/trips/tests/test_permissions.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+
+from .trip_factory import TripFactory
```

Comment:
Its better to determine what style of imports you use, relative or direct. 
Across all project.

Reply:
I'd advocate absolute imports everywhere.